<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerModule</class>
 <widget class="QWidget" name="PowerModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="mainLayout" rowstretch="1,1,1">
   <property name="leftMargin">
    <number>26</number>
   </property>
   <property name="topMargin">
    <number>26</number>
   </property>
   <property name="rightMargin">
    <number>26</number>
   </property>
   <property name="bottomMargin">
    <number>26</number>
   </property>
   <property name="spacing">
    <number>20</number>
   </property>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="pyrosLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="pyro1_frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">#pyro1_frame
{
	border-radius: 30%;
	border: 5px solid rgb(212,235,242);
	background-color: white;
	
} 


</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QGridLayout" name="pyro1Layout">
        <property name="topMargin">
         <number>6</number>
        </property>
        <item row="3" column="0">
         <widget class="QLCDNumber" name="Pyro1_maxAmp_LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLCDNumber" name="Pyro1_volt_LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLCDNumber" name="Pyro1_amp_LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="Pyro1_amp_label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Pyro 1 Current [A]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="Pyro1_maxAmp_label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Pyro 1 Max Current [A]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLCDNumber" name="Pyro1_watt_LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="Pyro1_volt_label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Pyro 1 Voltage [V]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="Pyro1_watt_label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Pyro 1 Power [W]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="pyro2_frame">
       <property name="styleSheet">
        <string notr="true">#pyro2_frame
{
	border-radius: 30%;
	border: 5px solid rgb(212,235,242);
	background-color: white;
	
} 


</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QGridLayout" name="pyro2Layout">
        <item row="1" column="1">
         <widget class="QLCDNumber" name="Pyro2_volt_LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="Pyro2_volt_label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Pyro 2 Voltage [V]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLCDNumber" name="Pyro2_amp_LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="Pyro2_maxAmp_label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Pyro 2 Max Current [A]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="Pyro2_amp_label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Pyro 2 Current [A]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLCDNumber" name="Pyro2_watt_LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="Pyro2_watt_label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Pyro 2 Power [W]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLCDNumber" name="Pyro2_maxAmp_LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="battery_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">#batteryFrame
{
	border-radius: 30%;
	border: 5px solid rgb(212,235,242);
	background-color: white;
	
} 


</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="batteryLayout" rowstretch="0,0,0,0">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="1" column="2">
       <widget class="QLCDNumber" name="ampBatLCD">
        <property name="styleSheet">
         <string notr="true">QFrame
{
	border: 0px, white;
}
</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="ampBatLabel">
        <property name="font">
         <font>
          <family>Segoe UI Black</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Battery Current [A]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLCDNumber" name="avgAmpBatLCD">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame
{
	border: 0px, white;
}</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="maxAmpBatCD">
        <property name="font">
         <font>
          <family>Segoe UI Black</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Battery Max Current [A]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLCDNumber" name="voltBatLCD">
        <property name="styleSheet">
         <string notr="true">QFrame
{
	border: 0px, white;
}</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLCDNumber" name="maxAmpBatLCD">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame
{
	border: 0px, white;
}</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="avgAmpBatLabel">
        <property name="font">
         <font>
          <family>Segoe UI Black</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Battery Avg Current [A]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="voltBatLabel">
        <property name="font">
         <font>
          <family>Segoe UI Black</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Battery Voltage [V]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="maxCapBatLabel">
        <property name="font">
         <font>
          <family>Segoe UI Black</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Battery Max Capacity [mAh]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLCDNumber" name="maxCapBatLCD">
        <property name="styleSheet">
         <string notr="true">QFrame
{
	border: 0px, white;
}</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QProgressBar" name="batBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar { height: 100px; border-radius: 20px;} 
QProgressBar::chunk { background-color: #138808;}</string>
        </property>
        <property name="value">
         <number>24</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="batBarLabel">
        <property name="font">
         <font>
          <family>Segoe UI Black</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Battery Level</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="voltageLayout" stretch="1,1">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QFrame" name="volt5V_frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">#voltage5Frame
{
	border-radius: 30%;
	border: 5px solid rgb(212,235,242);
	background-color: white;
	
} 


</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QGridLayout" name="voltage5Layout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item row="3" column="0">
         <widget class="QLCDNumber" name="avgVolt5LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="volt5Label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>5V Voltage [V]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="avgVolt5Label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>5V Avg Voltage [V]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLCDNumber" name="volt5LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="amp5Label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>5V Current [A]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="avgAmp5Label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>5V Avg Current [A]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLCDNumber" name="amp5LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLCDNumber" name="avgAmp5LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="volt3V3_frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">#voltage3Frame
{
	border-radius: 30%;
	border: 5px solid rgb(212,235,242);
	background-color: white;
	
} 


</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QGridLayout" name="voltage3Layout">
        <property name="topMargin">
         <number>6</number>
        </property>
        <item row="3" column="0">
         <widget class="QLCDNumber" name="avgVolt3LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="volt3Label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>3V3 Voltage [V]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="avgVolt3Label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>3V3 Avg Voltage [V]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLCDNumber" name="volt3LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="amp3Label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>3V3 Current [A]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="avgAmp3Label">
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>3V3 Avg Current [A]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLCDNumber" name="amp3LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLCDNumber" name="avgAmp3LCD">
          <property name="styleSheet">
           <string notr="true">QFrame
{
	border: 0px, white;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
